tracer/src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -603,6 +604,19 @@ private int WriteTags(ref byte[] bytes, int offset, in SpanModel model, ITagProc
                 }
             }
 
+            // Process tags will be sent only once per trace
+            if (model.IsFirstSpanInChunk && model.TraceChunk.ShouldPropagateProcessTags)
```

Comment:
don't forget to apply this only to the first span in payload (1st chunk only), like in these implementations
- https://github.com/DataDog/dd-trace-java/pull/9657
- https://github.com/DataDog/dd-trace-go/pull/4010